### PR TITLE
Move PromCon 2016 pages to /2016-berlin/

### PR DESCRIPTION
**Pull request by grobie**
The first PromCon is over, many more are to come. Let's prepare the site
to be able to host multiple conferences per year.

@juliusv

### CODE REVIEW

File: lib/helpers/default.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+include Nanoc::Helpers::LinkTo
+include Nanoc::Helpers::Rendering
+include Nanoc3::Helpers::Blogging
+include Nanoc3::Helpers::Tagging
+
+require 'nanoc/cachebuster'
+include Nanoc::Helpers::CacheBusting
+
+# TODO(ts): quick'n'dirty navigation hack. Build somthing nice.
```
Comment (juliusv) [inline lib/helpers/default.rb]: somthing -> something :P

File: lib/helpers/default.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    else
+      {
+        '/2016-berlin/' => '2016 - Berlin',
+      }
+    end
+
+  links.map do |link, text|
+    %(<li class="#{@item.identifier == link ? 'selected' : ''}"><a href="#{link}">#{text}</a></li>)
+  end.join("\n")
+end
```
Comment (juliusv) [inline lib/helpers/default.rb]: missing newline at end of file

File: content/2016-berlin/schedule.md [hunk 3]
```diff
@@ -1,12 +1,9 @@
 ---
-id: schedule
 title: Schedule
 ---
 
 ## PromCon 2016 Schedule
 
-This schedule is tentative and still subject to change.
```
Comment (juliusv) [inline content/2016-berlin/schedule.md]: After removing this paragraph, the headline is a bit too close to the table.

Comment (grobie) [inline content/2016-berlin/schedule.md]: Fixed via CSS.

File: lib/helpers/default.rb (hunk 2 above)
Comment (grobie) [inline lib/helpers/default.rb]: Fixed.

File: lib/helpers/default.rb (hunk 1 above)
Comment (grobie) [inline lib/helpers/default.rb]: ... I didn't play along and deleted my comment instead :P

**Comment by juliusv** [comment]
:+1:

**Comment by juliusv** [comment]
Somehow this already made it into "production", without it being merged... maybe because Travis deployed it? But the Travis config says it should only do it for master, hmmm...

**Pull request merged**